dnf-behave-tests/features/microdnf/install-xml-base.feature

```diff
@@ -0,0 +1,73 @@
+@no_installroot
+@desctructive
```

Comment:
You don't need the `@destructive` tag, `@no_installroot` already marks the tests as destructive.